cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/tasks/CassandraTaskExecutor.java

```diff
@@ -116,10 +116,12 @@ private CassandraTaskExecutor(
                 arguments,
                 uris,
                 ImmutableMap.<String, String>builder()
-                    .put("JAVA_HOME", javaHome)
-                    .put("JAVA_OPTS", "-Xmx" + heapMb + "M")
-                    .put("EXECUTOR_API_PORT", Integer.toString(apiPort))
-                    .build()))
+                        .put("LD_LIBRARY_PATH", "$MESOS_SANDBOX/libmesos-bundle/lib")
```

Comment:
I wouldn't expect dynamic `LD_LIBRARY_PATH`/`MESOS_NATIVE_JAVA_LIBRARY` to work as envvars. Shouldn't they be `export`s in the command itself?